README.md

```diff
@@ -61,3 +61,6 @@ POMDPs.initialstate(problem::Union{MDP, POMDP}, rng::AbstractRNG)
 ```
 which should return something of type State, where the argument problem is of type e.g. MDP{State, Action}
 
+To specify the type `A1` of the observation vector returned by the wrapper, you can implement the 
```

Comment:
I think the default way of changing OV should be in the environment constructor, so these docs should read:

"The type of the observation vector returned by an environment, `A1` may be specified as an argument to the environment constructor. The default for a particular `MDP` or `POMDP` may be specified by implementing `obsvector_type..."

Reply:
sounds good!